applications/ShapeOptimizationApplication/custom_utilities/response_functions/strain_energy_response_function.h

```diff
@@ -207,16 +201,16 @@ class StrainEnergyResponseFunction : ResponseFunction
 	}
 
 	// --------------------------------------------------------------------------
-	boost::python::dict GetGradient()
```

Comment:
i am approving, however as a hint i would give back a std::unordered_map

that would help you insulating from python. We'll discuss this separately though

Reply:
Thanks for the hint, this function is removed in PR #1804 . So i would keep this until we converged in the other, large PR.